pagopa/io-app
# chore(Cross): [IOAPPX-218] Add the new `Titillium Sans Pro` typeface to the codebase

## Short description
This PR adds the new `Titillium Sans Pro` ([official repo](https://github.com/chialab/titillium_pro))  font files to the codebase to ensure compatibility with the new `io-app-design-system` updates. 

## List of changes proposed in this pull request
- Add the new `Titillium Sans Pro` font files with relative license
- Link the new font files to iOS and Android builds
- Update `io-app-design-system` to `1.22.0`

## How to test
Run the app in the local environment. Try to change the font in some app components (e.g: `TabNavigator`) and check if everything is fine.
